### DISCUSSION TITLE
Combined additive and proportional error model

### DISCUSSION THREAD

**Discussion by mat194** [LAPKB/Pmetrics#227] [General]
Hi everyone! I noticed a code snippet on the GitHub page in the model section where model = combination(0.4, 3) #additive, proportional is used. However, when I try to use this in my project, I get a message saying "Combination error models are not supported yet." Is there a workaround for this issue? Additionally, is using combination error models widely different from a  proportional or additive error model?

**Comment by mnneely** [comment]
In this case the combination error model refers to the additional residual error not explained by the assay noise. The assay noise can be additive (C0 ≠ 0), multiplicative (C1 ≠ 0), parabolic (C2 ≠ 0) or cubic (C3 ≠ 0). The residual can be additive (lambda) or multiplicative (gamma). 

However, this is not exactly the same as for parametric modeling. In this case, the error is used as a weighting function for observations, such that the weight is 1/error, and error is (lambda + SD)^2 or (gamma * SD)^2, and SD = C0 + C1*obs + C2 * obs^2 + C3 * obs^3. Gamma or lamda generally are estimated and capture the residual error in the model.

**Reply by sumithmathew9** [reply]
I guess, in simple words, we can assume that the combined error model (additive and proportional error) is commonly used in Pmetrics.
